packages/office-ui-fabric-react/src/components/Shimmer/Shimmer.types.ts

```diff
@@ -0,0 +1,213 @@
+import * as React from 'react';
+import { IStyle, ITheme } from '../../Styling';
+import { IStyleFunctionOrObject } from '../../Utilities';
+
+export interface IShimmer {}
+
+/**
+ * Shimmer component props.
+ */
+export interface IShimmerProps extends React.AllHTMLAttributes<HTMLElement> {
+  /**
+   * Optional callback to access the IShimmer interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
+   */
+  componentRef?: (component: IShimmer | null) => void;
+
+  /**
+   * Sets the width of the shimmer wave wrapper in percentages relative to the containig parent element.
+   * @default 100%
+   */
+  widthInPercentage?: number;
```

Comment:
having both widthInPercentage and widthInPixel is confusing.

Reply:
Just have `width?: string | number;`

Reply:
Making the changes now... My initial thoughts were not to confuse the user on what to provide, a string or a number... but instead it's always a number and he chooses the specific one he needs.

Reply:
Fixed